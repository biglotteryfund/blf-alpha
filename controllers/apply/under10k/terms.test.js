/* eslint-env jest */
'use strict';
const terms = require('./terms');

test('default terms', function () {
    expect(terms('en')).toMatchSnapshot();
});

test('england specific terms', function () {
    const defaultTerms = terms('en');
    const englandTerms = terms('en', { projectCountry: 'england' });
    const scotlandTerms = terms('en', { projectCountry: 'scotland' });

    expect(englandTerms).toMatchSnapshot();

    expect(scotlandTerms).toStrictEqual(defaultTerms);
    expect(englandTerms).not.toStrictEqual(scotlandTerms);
});
