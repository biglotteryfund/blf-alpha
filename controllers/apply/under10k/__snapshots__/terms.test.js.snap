// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default terms 1`] = `
<ol class="o-nested-numbers">
  <li>
    <p>
      <strong>
        By submitting an application to The National Lottery Community Fund, the organisation named in the application (referred to as “you” in these Terms and Conditions) agrees, if awarded a grant, to:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        hold the grant on trust for The National Lottery Community Fund (referred to as “we” or “us”) and use it only for your project as described in your application or otherwise agreed with us, and only for expenditure incurred after the date of the grant award;
      </li>
      <li>
        provide us promptly with any information and reports we require about the project and its impact, both during and after the end of the project;
      </li>
      <li>
        act lawfully in carrying out your project, in accordance with best practice and guidance from your regulators, and follow any guidelines issued by us about the project or use of the grant;
      </li>
      <li>
        acknowledge National Lottery funding using the common lottery branding in accordance with the relevant brand guidelines
      </li>
      <li>
        hold the grant in a UK based bank or building society account which satisfies our requirements as set out in guidelines and requires at least two unconnected people to approve all transactions and withdrawals;
      </li>
      <li>
        immediately return any part of the grant that is not used for your project or which constitutes unlawful state aid;
      </li>
      <li>
        where your project involves working with children, young people or vulnerable adults, adopt and implement an appropriate written safeguarding policy, obtain written consent from legal carers or guardians and carry out background checks for all employees, volunteers, trustees or contractors as required by law or our guidelines;
      </li>
      <li>
        comply with data protection laws and obtain the consent of your beneficiaries for us and you to receive and process their personal information and contact them;
      </li>
      <li>
        keep accurate and comprehensive records about your project both during the project and for seven years afterwards and provide us on request with copies of those records and evidence of expenditure of the grant, such as original receipts and bank statements;
      </li>
      <li>
        allow us and/or the Comptroller and Auditor General reasonable access to your premises and systems to inspect project and grant records;
      </li>
      <li>
        The National Lottery Community Fund publicising and sharing information about you and your project including your name and images of project activities. You hereby grant us a royalty free licence to reproduce and publish any project information you give us. You will let us know when you provide the information if you don’t have permission for us to use it in this way; and
      </li>
      <li>
        if your project is being delivered in Wales, enable people to engage in both Welsh and English, treating both languages equally. Welsh speakers must be able to access information and services in Welsh and all materials must be produced bilingually.
      </li>
    </ol>
  </li>
  <li>
    <p>
      <strong>
        You acknowledge that we are entitled to suspend or terminate the grant and/or require you to repay all or any of the grant in any of the following situations. You must let us know if any of these situations have occurred or are likely to occur:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        You use the grant in any way other than as approved by us or fail to comply with any of these Terms & Conditions.
      </li>
      <li>
        You fail to make good progress with your project or are unlikely in our view to complete the project or achieve the objectives agreed with us.
      </li>
      <li>
        You have match funding for the project withdrawn or receive duplicate funding for the same project costs as funded by the grant.
      </li>
      <li>
        You provide us with false or misleading information either on application or after award of the grant, act dishonestly or are under investigation by us, a regulatory body or the police, or if we consider for any other reason that public funds are at risk or you do anything to bring us or the National Lottery into disrepute.
      </li>
      <li>
        You enter into, or in our view are likely to enter into, administration, liquidation, receivership, dissolution or, in Scotland, have your organisation’s estate sequestrated.
      </li>
    </ol>
  </li>
  <li>
    <p>
      <strong>
        You acknowledge that:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        the grant is for your use only and we may require you to pay us a  share of any proceeds from disposal of assets purchased or enhanced with the grant;
      </li>
      <li>
        we will not increase the grant if you spend more than the agreed budget and we can only guarantee the grant as long as the National Lottery continues to operate and we receive sufficient funds from it;
      </li>
      <li>
        the grant is not consideration for any taxable supply for VAT purposes;
      </li>
      <li>
        we have no liability for any costs or consequences incurred by you  or third parties that arise directly or indirectly from the project, nor from non-payment or withdrawal of the grant, save to the extent required by law;
      </li>
      <li>
        these Terms and Conditions will continue to apply for one year after  the grant is paid or until the project has been completed, whichever is later. Clauses 1.2, 1.4, 1.6, 1.9, 1.10, 1.11 and 3.4 shall survive expiry of these Terms and Conditions; and
      </li>
      <li>
        If the application and grant award are made electronically, the agreement between us shall be deemed to be in writing and your online acceptance of these Terms and Conditions shall be deemed to be a signature of that agreement.
      </li>
    </ol>
  </li>
</ol>
`;

exports[`england specific terms 1`] = `
<ol class="o-nested-numbers">
  <li>
    <p>
      <strong>
        By submitting an application to The National Lottery Community Fund, the organisation named in the application (referred to as “you” in these Terms and Conditions) agrees, if awarded a grant, to:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        hold the grant on trust for The National Lottery Community Fund (referred to as ‘we’ or ‘us’) and use it only for your project as described in your application or otherwise agreed with us, and only for expenditure incurred after the date of your grant award;
      </li>
      <li>
        provide us promptly with any information and reports we require about the project and its impact, both during and after the end of the project;
      </li>
      <li>
        act lawfully in carrying out your project in accordance with best practice and guidance from your regulators, and follow any guidelines issued by us about the project or use of the grant and let us know promptly about any fraud, other impropriety, mismanagement or misuse in relation to the grant;
      </li>
      <li>
        acknowledge National Lottery funding using our logo in accordance with the relevant  guidelines for recognising your grant, which can be found on our website
        <a href="https://www.tnlcommunityfund.org.uk/">
          https://www.tnlcommunityfund.org.uk/
        </a>
        ;
      </li>
      <li>
        hold the grant in a UK based bank or building society account which satisfies our requirements as set out in guidelines and requires at least two unconnected people to approve all transactions and withdrawals;
      </li>
      <li>
        immediately return any part of the grant that is not used for your project or which constitutes unlawful state aid;
      </li>
      <li>
        where your project involves working with children, young people or vulnerable adults, adopt and implement an appropriate written safeguarding policy, obtain written consent from legal carers or guardians and carry out background checks for all employees, volunteers, trustees or contractors as required by law or our guidelines;
      </li>
      <li>
        work with the evaluators appointed for our Covid-19 funding in order to share evidence, data and learning. This will include taking part in the following evaluation activities:
        <ul>
          <li>
            a short survey about your experience of the Fund's service to you (customer satisfaction)
          </li>
          <li>
            an end of grant survey, this will include questions on: the number and type of activities you’ve run using the funding, the number of beneficiaries reached due to the funding, number of volunteers mobilised due to the funding, the difference the funding made, what you achieved, who you worked with and if this funding enable access to other funds.
          </li>
          <li>
            send a survey to your volunteers about their experience
          </li>
          <li>
            a smaller sample of grant holders will be invited to take part in more in-depth, qualitative research
          </li>
          <li>
            sharing learning and insight for other charities
          </li>
        </ul>
      </li>
      <li>
        comply with data protection laws and obtain the consent of your beneficiaries for us and you to receive and process their personal information and contact them in connection with the project;
      </li>
      <li>
        keep accurate and comprehensive records about your project both during the project and for seven years afterwards and provide us on request with copies of those records and evidence of expenditure of the grant, such as original receipts and bank statements;
      </li>
      <li>
        allow us and/or the Comptroller and Auditor General reasonable access to your premises and systems to inspect project and grant records;
      </li>
      <li>
        The National Lottery Community Fund publicising and sharing information about you and your project including your name and images of project activities. You hereby grant us a royalty free licence to reproduce and publish any project information you give us. You will let us know when you provide the information if you don’t have permission for us to use it in this way; and
      </li>
      <li>
        if your project is being delivered in Wales, enable people to engage in both Welsh and English, treating both languages equally. Welsh speakers must be able to access information and services in Welsh and all materials must be produced bilingually.
      </li>
    </ol>
  </li>
  <li>
    <p>
      <strong>
        You acknowledge that we are entitled to suspend or terminate the grant and/or require you to repay all or any of the grant in any of the following situations. You must let us know if any of these situations have occurred or are likely to occur:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        You use the grant in any way other than as approved by us or fail to comply with any of these Terms and Conditions.
      </li>
      <li>
        You fail to make good progress with your project or are unlikely in our view to complete the project or achieve the objectives agreed with us.
      </li>
      <li>
        You have match funding for the project withdrawn or receive or fail to declare any duplicate funding for the same project costs as funded by the grant.
      </li>
      <li>
        You provide us with false or misleading information either on application or after award of the grant, act dishonestly or are under investigation by us, a regulatory body or the police, or if we consider for any other reason that public funds are at risk or you do anything to bring us or the National Lottery into disrepute.
      </li>
      <li>
        You enter into, or in our view are likely to enter into, administration, liquidation, receivership, dissolution or, in Scotland, have your organisation’s estate sequestrated.
      </li>
      <li>
        You receive any grant money incorrectly either as a result of an administrative error or otherwise. This includes where You are paid in error before You have complied with your obligations under these terms and conditions and Offer Letter. Any sum, which falls due under this paragraph 2.6, shall fall due immediately. If the You fail to repay the due sum immediately, or as otherwise agreed with us, the sum will be recoverable summarily as a civil debt.
      </li>
    </ol>
  </li>
  <li>
    <p>
      <strong>
        You acknowledge that:
      </strong>
    </p>
    <ol class="o-nested-numbers">
      <li>
        the grant is for your use only and we may require you to pay us a share of any proceeds from disposal of assets purchased or enhanced with the grant;
      </li>
      <li>
        we will not increase the grant if you spend more than the agreed budget and we can only guarantee the grant as long as The National Lottery continues to operate and we receive sufficient funds from it;
      </li>
      <li>
        the grant is not consideration for any taxable supply for VAT purposes;
      </li>
      <li>
        we have no liability for any costs or consequences incurred by you or third parties that arise directly or indirectly from the project, nor from non-payment or withdrawal of the grant, save to the extent required by law;
      </li>
      <li>
        these Terms and Conditions will continue to apply, unless terminated  earlier in accordance with Clause 2, for one year after the grant is paid or until the project has been completed, whichever is later. Clauses 1.2, 1.3, 1.4, 1.6, 1.8, 1.9, 1.10, 1.11, 1.12, 1.13, 2.1, 2.3, 2.4, 2.5, 2.6, 3.1, 3.4, 3.5 and 3.6 shall survive expiry of these Terms and Conditions; and
      </li>
      <li>
        if the application and grant award are made electronically, the agreement between us shall be deemed to be in writing and your online acceptance of these Terms and Conditions shall be deemed to be a signature of that agreement.
      </li>
    </ol>
  </li>
</ol>
`;
